### PR TITLE
Create ui-c8y-1020-21-1-updating-karma-6973-graft-release-cd-6980.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60560](https://cumulocity.atlassian.net/browse/MTM-60560)
Original PR: [6980](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6980)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-21-1-updating-karma-6973-graft-release-cd-6980.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: updating karma (#6973) [GRAFT][release/cd] (#6980)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-21-1-updating-karma-6973-graft-release-cd-6980.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Update Karma to version 6.4.0
```

File: content/change-logs/application-enablement/ui-c8y-1020-21-1-updating-karma-6973-graft-release-cd-6980.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60560
+version: 1020.21.1
+---
+updating karma (#6973) [GRAFT][release/cd] (#6980)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-21-1-updating-karma-6973-graft-release-cd-6980.md:17]: The Claude AI suggested to use the following body:
```suggestion
As part of ongoing efforts to keep the Cumulocity IoT platform up-to-date with the latest dependencies, Karma has been updated from version 6.3.20 to version 6.4.0. This update brings the latest improvements and bug fixes in Karma, ensuring better stability and performance for running frontend unit tests. The update is not expected to have any direct impact on end users or existing applications, but it contributes to the overall health and maintainability of the Cumulocity IoT platform.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@janhommes Please review and approve

**Pull request closed without merging**